aten/src/ATen/native/layer_norm.cpp

```diff
@@ -165,7 +165,60 @@ Tensor quantized_layer_norm_impl(
     X.suggest_memory_format());
 
   if (M > 0) {
-    quantized_layer_norm_stub(kCPU, X, gamma, beta, M, N, eps, &Y);
+    bool affine_per_channel = false;
+    int num_channels = 1; // not relevant for LayerNorm
+    int num_groups = 1; // not relevant for LayerNorm
+    quantized_normalize_stub(kCPU, X, gamma, beta, affine_per_channel,
+        num_channels, num_groups, M, N, eps, &Y);
+  }
+  return Y;
+}
+
+Tensor quantized_group_norm_impl(
```

Comment:
Is there a reason why this isn't defined in the native/quantized/cpu folder?

Reply:
It was mostly convenience of reusing `_prepare_layer_norm_inputs` which is used by LayerNorm.  Would `native/quantized/cpu` be a better place for this / is it worth refactoring? I'm flexible

Reply:
I think it would be good to have all the quantized ops live in the same folder. You should be able to call `_prepare_layer_norm_inputs` from there. i think we do something similar in qpool.cpp where we reuse the output shape computation logic.

Reply:
fixed this in https://github.com/pytorch/pytorch/pull/36835, hope that's ok!